tests/baselines/reference/propertyAssignmentOnImportedSymbol.errors.txt

```diff
@@ -0,0 +1,11 @@
+tests/cases/conformance/salsa/bug24658.js(1,10): error TS2440: Import declaration conflicts with local declaration of 'hurk'.
+
+
+==== tests/cases/conformance/salsa/mod1.js (0 errors) ====
+    export var hurk = {}
+==== tests/cases/conformance/salsa/bug24658.js (1 errors) ====
+    import { hurk } from './mod1'
+             ~~~~
+!!! error TS2440: Import declaration conflicts with local declaration of 'hurk'.

```

Comment:
Hm. Should we make this merge work in JS, rather than making it an error? We already allow import merges when the kinds don't conflict, and JS merges are like a merge-when-the-kinds-to-conflict.

Reply:
Filed #26912 to track merging imports and assignment declarations